moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -524,135 +524,139 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
 
 static void wrap_move_group_interface()
 {
-  bp::class_<MoveGroupInterfaceWrapper, boost::noncopyable> MoveGroupInterfaceClass(
+  bp::class_<MoveGroupInterfaceWrapper, boost::noncopyable> move_group_interface_class(
```

Comment:
This is a weird one. IMO since `MoveGroupInterfaceClass` is actually defining a python class, it should be stylized as a class name.

Reply:
IMO, this is used for the convenience of exposing c++ class to python, the name matters for its first argument, it can be acceptable to be lower_case.

Reply:
@BryceStevenWilley, you are right, this is defining a python class. However, formally it's doing so using a C++ variable. @RoboticsYY is right, that this variable name doesn't really matter - it's not exposed anywhere (could be even `foo`).